### ISSUE TITLE
lgtm.com CI failing because libostree is too old

### ISSUE THREAD

**Issue by smcv** [flatpak/flatpak#4016]
lgtm.com appears to build on Ubuntu 19.10 'eoan', which has a libostree that is too old for Flatpak git master:

https://lgtm.com/projects/g/flatpak/flatpak/logs/languages/lang:cpp
```
[2020-12-07 18:30:58] [build-err] configure: error: Package requirements (ostree-1 >= 2020.8) were not met:
[2020-12-07 18:30:58] [build-err] Package dependency requirement 'ostree-1 >= 2020.8' could not be satisfied.
[2020-12-07 18:30:58] [build-err] Package 'ostree-1' has version '2019.3', required version is '>= 2020.8'
```

(No, I don't know why they're using a short-term-supported Ubuntu version that is already EOL, when a newer LTS version is available...)

To fix the CI I think it'll be necessary to either disable LGTM analysis until they provide a newer container (there is currently no suitable version of Ubuntu, although libostree 2020.8 is queued for inclusion in Ubuntu 21.04 'hirsute'), or make the use of the new libostree summary features optional, or build libostree from source as part of the LGTM build (if that's possible).

**Comment by mwleeds** [comment]
>or build libostree from source as part of the LGTM build (if that's possible).

That's what I tried to do in #3970 but it didn't work for reasons I haven't had time to investigate.

**Comment by mwleeds** [comment]
>To fix the CI I think it'll be necessary to either disable LGTM analysis until they provide a newer container

Seems reasonable but I don't have enough permissions to do it.

**Comment by mwleeds** [comment]
This will be fixed by #4034 I think
